algorithms/unit_tests/TestBinSortA.hpp

```diff
@@ -219,6 +219,9 @@ void test_sort_integer_overflow() {
 }  // namespace BinSortSetA
 
 TEST(TEST_CATEGORY, BinSortGenericTests) {
+#if defined(KOKKOS_COMPILER_CRAY_LLVM) && defined(KOKKOS_ENABLE_OPENMPTARGET)
```

Comment:
```suggestion
// FIXME KOKKOS_COMPILER_CRAY_LLVM
#if defined(KOKKOS_COMPILER_CRAY_LLVM) && defined(KOKKOS_ENABLE_OPENMPTARGET)
```

Reply:
The suggested FIXME marker seems inconsistent with our current practice

Reply:
The FIXME marker should start with `FIXME_OPENMPTARGET - ...` . I will make the changes.